qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -137,7 +137,16 @@ public void test50StartAndStop() throws Exception {
         // cleanup from previous test
         rm(installation.config("elasticsearch.keystore"));
 
-        Archives.runElasticsearch(installation, sh);
+        try {
```

Comment:
Would it make sense to just bake this into `waitForElasticsearch()`?

Reply:
The pid file will not always be at the same location. We could but it would take significantly more work to make sure we always track the pid file in the `installation` so we can reliably reference it

Reply:
That's fine. If we expect this to be mostly temporary, or only really helpful for this test, then let's press on as-is.